core-client/Cargo.toml

```diff
@@ -20,15 +20,19 @@ categories = [
 
 [dependencies]
 failure = "0.1"
-futures = "~0.1.6"
+futures = "0.1.26"
+hyper = "0.12"
```

Comment:
Should we make the transports features? Maybe worth a separate issue.

Reply:
https://github.com/paritytech/jsonrpc/issues/431